packages/wallets/src/components/ProgressBar/ProgressBar.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import './ProgressBar.scss';
+
+type TProps = {
+    is_transition?: boolean;
+    active_index: number;
+    indexes: Array<number>;
+    setActiveIndex: React.Dispatch<React.SetStateAction<number>>;
+};
+
+const ProgressBar: React.FC<TProps> = ({ is_transition, active_index, indexes, setActiveIndex }) => {
+    return (
+        <div className='wallets-progress-bar'>
+            {indexes.map(st => {
```

Comment:
@thisyahlen-deriv please rename this to the more meaningful one
```suggestion
            {indexes.map(idx => {
```